package.json

```diff
@@ -24,7 +24,7 @@
     "touch": "0.0.3"
   },
   "devDependencies": {
-    "coffee-script": "^1.9.1"
+    "coffee-script": "1.9.1"
```

Comment:
Preferable to avoid 1.9.1 because of watch file sync error (see 1.9.2 changelog - "Fixed a watch mode error introduced in 1.9.1 when compiling multiple files with the same filename.")